docs/guides/data/storing_output.md

```diff
@@ -100,7 +100,7 @@ even after your job ends, and can be used again in future jobs.
 You can view the saved output of a job using the `floyd output` command:
 
 ```bash
-$ floyd output floydhub/projects/quick-start/1/
+$ floyd output mckay/projects/quick-start/1/
```

Comment:
is the trailing slash required?

Reply:
Good catch. Didn't see this comment until after the merge. I have fixed this now.